src-gui/src/renderer/rpc.ts

```diff
@@ -417,29 +435,169 @@ export async function getMoneroAddresses(): Promise<GetMoneroAddressesResponse>
   return await invokeNoArgs<GetMoneroAddressesResponse>("get_monero_addresses");
 }
 
+export async function getRestoreHeight(): Promise<GetRestoreHeightResponse> {
+  return await invokeNoArgs<GetRestoreHeightResponse>("get_restore_height");
+}
+
+export async function setMoneroRestoreHeight(
```

Comment:
@Matroskine We want this to take an enum { year: u32, month: u32, day: u32 } instead of a string

Needs to be changed in request.rs as well